x-pack/plugins/apm/public/components/routing/service_detail/index.tsx

```diff
@@ -135,6 +135,8 @@ export const serviceDetail = {
               t.partial({
                 traceId: t.string,
                 transactionId: t.string,
+                comparisonEnabled: toBooleanRt,
```

Comment:
I just noticed that on line 88 we have:
```ts
refreshPaused: t.union([t.literal('true'), t.literal('false')]),
```

should that be changed to:
```ts
refreshPaused: toBooleanRt,
```

?

(asking because I'm wondering why we have to different ways of writing this)

Reply:
Yes, we usually use `toBooleanRt`.

Reply:
Let's be careful here. toBooleanRt converts to a boolean, the other one should leave the string literals in place. There might be a reason for that.